### PR TITLE
Add autodeploy, improve DEVELOPERS.md, and better expressions for build.bat

### PR DESCRIPTION
**Pull request by 7k5x**
-  Added a CD script to auto release `wsldl` with the download count badge ***only when you push the tag by `git push --tags`***. 
 - Added instructions for autobuild in `DEVELOPERS.md`
 - `build.bat` had some expression improvements.
 - [Suggestion] How about adding `call :clean` to the `all` option in `build.bat` to clean the `.syso` files after building?

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -36,16 +36,16 @@ exit /b
 cd /d %~dp0
 mkdir tools >NUL 2>&1
 echo Downloading rsrc...
-curl -sSfL https://github.com/akavel/rsrc/releases/download/v0.10.2/rsrc_windows_amd64.exe -o tools\rsrc.exe
-echo Compiling All Resources...
+curl -sSfLO https://github.com/akavel/rsrc/releases/download/v0.10.2/rsrc_windows_amd64.exe -o tools\rsrc.exe
```
Comment (yuk7) [inline build.bat:39]: I don't think the "O" option can be used with the "o" option...

**Review by yuk7 — CHANGES_REQUESTED** [review]
Thank you very much!
but, could you please check about build.bat?

I don't think it's appropriate to do post-build cleanup.
I think most Makefiles will keep the build-time intermediate files.

File: build.bat (hunk 1 above)
Comment (7k5x) [reply build.bat:39]: Oops! My bad😭Sorry..I'll fix it right away....

**Comment by 7k5x** [comment]
A postbuild cleanup was just a suggestion, it wasn't commited😇

**Review by yuk7 — APPROVED** [review]
Ok

**Comment by yuk7** [comment]
haha, thank you very much!

**Pull request merged**